packages/dx-react-grid-bootstrap3/src/templates/table-group-cell/cell.test.jsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+import { setupConsole } from '@devexpress/dx-testing';
+import { Cell } from './cell';
+
+describe('TableCell', () => {
```

Comment:
I think we should add one more describe to all tests with a 'TableGroupCell' message.

> TableGroupCell
>> Cell
>> Context
>> Icon